### ISSUE TITLE
Add new and restore old options to get mechanical keycap style from keyviz 1.0.6

### ISSUE THREAD

**Issue by frank82vt** [mulaRahul/keyviz#287]
**Problem**

1.-The mechanical keycap options from keyviz 1.0.6, were removed in later versions of keyviz
In the video you can see the mechanical keycaps of keyviz version 1.0.6 and 2.0.0a2 on the bottom and middle right side of the screen respectively. For comparison purposes, the keycaps of version 2.0.0a2 were configured to look similar to those of version 1.0.6. The mechanical keycap model, style, size options of keyviz version 1.0.6 are elegant, have good quality and appearance but were removed from keyviz in later versions.

2.-It is not possible to configure the size of the keycaps in version 2.0.0a2 of keyviz
In the video you can see that, since it is not possible to configure the size of the keycaps in the version of keyviz 2.0.0a2, these keycaps have a small standard size that is lengthened to allow the display of the text, even when it is shortened or the font is reduced. Meanwhile, version 1.0.6 keycaps are larger and retain their shape without lengthening, since their configured size is large enough to contain the keycap text. For example, see the case of the keycaps page down, home in the video

3.- Some keycap icons are missing in recent versions of keyviz
For example, in the video see the case of the space and menu ≣ keys (keycap apps in version 1.0.6 in the video). Each keycap should have the option to have an icon whenever it exists.

**Version**: keiviz v2.0.0a2

**Solution**
It is requested to add new and restore old necessary options in keyviz to obtain the model, style, size of the mechanical keys of keyviz version 1.0.6.

+In keyviz, settings, keycap, add:
-style section, isometric option.[i]
-model section, mechanical keycap model same as version 1.0.6
-size section, from 32px to 48px. This will allow the keycap not to have to be lengthened to contain text even when it is small and cropped [i]

Note:
i).There was this option in v1.0.6 and it was removed in later versions, see fig 1 below

**Alternatives**
Alternatively the specific keycap model can be selected in the presets section as a nested option, for example:
+In keyviz, settings, keycap, add:
 -presets section, mechanical, mechanical model like keyviz version 1.0.6.
 
 althoug personally i prefer the solution described in solution section above

**video**
watch video

**screenshots**
![image](https://github.com/mulaRahul/keyviz/assets/84551251/e6d39c26-70e0-4b6e-8926-009613598149)
fig 1

https://github.com/mulaRahul/keyviz/assets/84551251/fc3d34db-22ba-4a35-b1dd-656eec03a42f

**Comment by mulaRahul** [comment]
Have you checked the **Plastic** preset?

**Comment by jimmckeeth** [comment]
I just switched back to 1.0.6 since I liked the keycap design better.

**Comment by frank82vt** [comment]
> That's actually a good suggestion. I myself like the previous version as well. Will make the change!
Thanks, its good to know that.

**Comment by frank82vt** [comment]
> Have you checked the **Plastic** preset?
Yes i checked plastic preset in keiviz v2.0.0a2 but by now i didnt stumbled with a particular problem with that option.
